### DISCUSSION TITLE
Get updates inside the game

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22524] [Ideas] [answered]
Updates inside the game seems good idea, probably.

Downloading updates inside the game and the updates will be applied after restarting the game. I've saw it in some apps but not sure if it's a good idea in games.

In that case, We update lovers don't need to wait for the update to be release in the github. We can just update the game whenever changes happen.

**Comment by papertek** [comment, marked as answer]
is this for mobile? or..?

because the game already does this

**Reply by mk56-spn** [reply]
probably mobile or linux, yes, not sure if mac autoupdates either

**Reply by Shirou27** [reply]
For mobile

**Reply by mk56-spn** [reply]
I reread your post. No "updates" happen that don't immediately get released on GitHub. Merges happen, but those should not update into mobile directly for a plethora of reasons..builds, merges needing other merged to work properly etc..eitherway this suggestion doesn't make a lot of sense since app store / play store releases are coming soon TM afaik
